contracts/ERC721.sol

```diff
@@ -1239,6 +1241,7 @@ contract TradeTrustERC721 is ERC721MintableFull, IERC721Receiver {
   function destroyToken(uint256 _tokenId) public onlyMinter {
     require(ownerOf(_tokenId) == address(this), "Cannot destroy token: Token not owned by token registry");
     _burn(_tokenId);
+    emit TokenBurnt(_tokenId);
   }
 
```

Comment:
Should we also have an events for receiving token? That will allow web app to pick up the events, helpful if we want to improve the endorsement chain view